topics/profiles-call-tree.yaml

```diff
@@ -0,0 +1,13 @@
+topic: profiles-call-tree
+pipeline: profiles
+description: Function data extracted from profiles for Snuba
+services:
+  producers:
+    - getsentry/vroom
+  consumers:
+    - getsentry/snuba
+schemas:
+  - version: 1
+    compatibility_mode: none
+    type: json
+    resource: profile-functions.v1.schema.json
```

Comment:
examples key is missing here. I'd add 1-2 per schema. Once you bump the schema repo in Snuba, your message processor will also be running against those examples in tests